### PR TITLE
Srxio

### PR DESCRIPTION
**Pull request by licode**
Interface for srx is updated.

### CODE REVIEW

File: pyxrf/model/draw_image.py [hunk 1]
```diff
@@ -109,7 +109,7 @@ class DrawImageAdvanced(Atom):
     items_in_selected_group = List()
 
     scale_opt = Str('Linear')
-    color_opt = Str('Oranges')
+    color_opt = Str('jet')
```
Comment (ericdill) [inline pyxrf/model/draw_image.py]: wait what. why jet?

File: pyxrf/model/fileio.py [hunk 2]
```diff
@@ -300,7 +309,10 @@ def get_spectrum(self, mask=None):
 def file_handler(working_directory, file_name):
     # send information on GUI level later !
     try:
-        return read_hdf_APS(working_directory, file_name)
+    	if beamline_name == 'HXN':
+                return read_hdf_APS(working_directory, file_name, spectrum_cut=2000)
+       	if beamline_name == 'SRX':
+                return read_hdf_APS(working_directory, file_name, spectrum_cut=2000)
```
Comment (ericdill) [inline pyxrf/model/fileio.py]: your code formatting here is off

File: pyxrf/model/fileio.py [hunk 3]
```diff
@@ -1443,61 +1400,74 @@ def get_scaler_list_hxn(all_keys):
...
+        hdr = db[runid]
+        print('Loading data from database.')
+        # fields = [] to be used later
+        data = get_table(hdr)
+
+        start_doc = hdr['start']
+        datashape = start_doc['shape']
+        datashape = [datashape[1], datashape[0]]  # vertical first, then horizontal
+        fly_type = None
+    	snake_scan = start_doc.get('snaking')
```
Comment (ericdill) [inline pyxrf/model/fileio.py]: spacing is off here too

**Comment by licode** [comment]
sorry. I merged it fast as I want to do conda build on new master branch.

File: pyxrf/model/fileio.py (hunk 2 above)
Comment (licode) [inline pyxrf/model/fileio.py]: It is weird because everything looks fine in atom.

**Pull request merged**